src/components/ManageAccountsDrawer/ManageAccountsDrawer.tsx

```diff
@@ -1,42 +1,65 @@
-import { ethChainId } from '@shapeshiftoss/caip'
-import { useEffect, useMemo, useState } from 'react'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { useCallback, useMemo, useState } from 'react'
 import { assertUnreachable } from 'lib/utils'
 
 import { DrawerWrapper } from './components/DrawerWrapper'
 import { ImportAccounts } from './components/ImportAccounts'
+import { SelectChain } from './components/SelectChain'
 
 export type ManageAccountsDrawerProps = {
   isOpen: boolean
   onClose: () => void
 }
 
-type ManageAccountsStep = 'manageAccounts' | 'selectChain' | 'ledgerOpenApp' | 'importAccounts'
+type ManageAccountsStep = 'selectChain' | 'ledgerOpenApp' | 'importAccounts'
 
 export const ManageAccountsDrawer = ({ isOpen, onClose }: ManageAccountsDrawerProps) => {
-  const [step, setStep] = useState<ManageAccountsStep>('manageAccounts')
+  const [step, setStep] = useState<ManageAccountsStep>('selectChain')
+  const [selectedChainId, setSelectedChainId] = useState<ChainId | null>(null)
 
-  // TEMP: Set the initial step to 'importAccounts' until we have the other steps implemented
-  useEffect(() => {
-    setStep('importAccounts')
-  }, [])
+  const isLedger = false
```

Comment:
@0xApotheosis 👀

Reply:
Any reason to monkey patch it to false vs. checking for actual `IsLedger()` ?

Reply:
Even though `ledgerOpenApp` isn't implemented yet in this PR, still makes sense to do the regular check vs. a monkey patch?